### ISSUE TITLE
How can i post-process inline styles

### ISSUE THREAD

**Issue by teamrun** [vuejs/vue-loader#689]
Here is the case: i hope to be able to process inline style wrote in template as well as `<style>` declearations.

```
<template>
    <div class="wraper" :style="{'justify-content': justiVal}">
        <text>inline style process</text>
    </div>
</template>

<style>
    // i know postcss can handle these in <style> tag
    .wraper{
        display: flex;
    }
</style>

/*
    how can i process inline style(dynamic binding style or static style) :style="{'justify-content': justiVal}" into this:
    style: { '-webkit-justify-content': justiVal, 'justify-content': justiVal}
*/

<script>
    module.exports = {
        data: function(){
            return {justiVal: 'flex-end'}
        }
    }
</script>
```

**Comment by MrRaindrop** [comment]
I got the same problem, and why I didn't use style tag for this line of style is that the style data is always dynamic and from server fetching. What I did to solve this problem is to handle the style in rendering function and adding the vendors in runtime. This is a bit of hack and performance unendurable. What can I do to solve this kind of processing of styles, in compiling time, instead of runtime ?
